src/views/LoadManagementConfig.vue

```diff
@@ -351,6 +390,15 @@ export default {
 					}, {});
 			},
 		},
+		counterOptions() {
+			var myOptions = [];
+			for (const element of Object.values(this.componentConfigurations)) {
+				if (this.isComponentType(element.type, "counter")) {
+					myOptions.push({ value: element.id, text: element.name });
+				}
+			}
+			return myOptions;
```

Comment:
Es wäre gut, wenn die Optionen alphabetisch sortiert werden.
```suggestion
			return myOptions.sort((a, b) => {
				if (a.text == b.text) {
					return 0;
				}
				return a.text > b.text ? 1 : -1;
			});
```